packages/context/src/objects/objects.ts

```diff
@@ -43,6 +43,51 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
+export type RegexValidation = {
+    regex: string;
+    errorMessage: string;
+};
+
+export type SelectOption = {
+    label: string;
+    value: string;
+};
+
+/**
+ * Represents an object action inputProperty object.
+ */
+export type FormioInput = {
+    label: string;
+    type: string;
+    key: string;
+    initialValue?: unknown;
+    input?: boolean;
+    placeholder?: string;
+    description?: string;
+    tooltip?: string;
+    prefix?: string;
+    suffix?: string;
+    showCharCount?: boolean;
+    readOnly?: boolean;
+    isMultiLineText?: boolean;
+    /**
+     * An array of values required for select options.
```

Comment:
This should go with `data.values`, not `data`.

Reply:
Done.